### PR TITLE
Feature/ahs local sim

### PR DESCRIPTION
**Pull request by maolinml**
*Issue #, if available:*

*Description of changes:*

Add the `inputs` argument as a placeholder into the ahs local simulator such that it can be run on the sdk

*Testing done:*

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

#### General

- [ ] I have read the [CONTRIBUTING](https://github.com/aws/amazon-braket-sdk-python/blob/main/CONTRIBUTING.md) doc
- [ ] I used the commit message format described in [CONTRIBUTING](https://github.com/aws/amazon-braket-sdk-python/blob/main/CONTRIBUTING.md#commit-your-change)
- [ ] I have updated any necessary documentation, including [READMEs](https://github.com/aws/amazon-braket-sdk-python/blob/main/README.md) and [API docs](https://github.com/aws/amazon-braket-sdk-python/blob/main/CONTRIBUTING.md#documentation-guidelines) (if appropriate)

#### Tests

- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have checked that my tests are not configured for a specific region or account (if appropriate)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by maolinml** [comment]
@krneta @peterkomar-aws Sorry guys, just noticed that I need to add this argument so that the local simulator can run properly. This should have been there in the previous PR.

**Comment by peterkomar-aws** [comment]
I was able to run the local simulator successfully using the current `feature/ahs` branch (commit e6a8005e2f5900a8b5dce3ddcdeaa01fe2e3586b) with the `feature/neutral-atom-simulator-solver` branch of the local simulator (commit 6dfdec23d64f10c1f099d26aaf6a8b343666e90c ).

* What is the error you are seeing? 
* Which branch of the local simulator are you using?

**Review by peterkomar-aws — APPROVED** [review]

File: src/braket/devices/local_simulator.py [hunk 1]
```diff
@@ -193,8 +193,17 @@ def _(
 
 
 @_run_internal.register
-def _(program: AnalogHamiltonianSimulation, simulator: BraketSimulator, shots: Optional[int] = None, *args, **kwargs):
+def _(
+    program: AnalogHamiltonianSimulation, 
+    simulator: BraketSimulator, 
+    shots: Optional[int] = None, 
+    inputs: Optional[Dict[str, float]] = None,
```
Comment (krneta) [inline src/braket/devices/local_simulator.py:200]: You shouldn't need to declare inputs, especially if they're not used. If they are provided, they should be in args/kwargs.

**Comment by maolinml** [comment]
Hi @peterkomar-aws, after some conversation with Aaron and Milan, it turns out that the more elegant solution to support the neutral atom local simulator which, unlike other simulators, does not have the `input` arguments, is to 
1. pass `inputs` as a keyword argument as we did in this PR
2. Add `*args` and `**kwargs` in the neutral atom local sim, as in this PR https://github.com/aws/amazon-braket-default-simulator-python/pull/121/commits/d232f945b191c3dcc78ff4e7ee769126bdb7646a

Hope this clears some doubts.

**Comment by maolinml** [comment]
@krneta @ajberdy If you have time, could you take a quick look at this?

**Review by krneta — APPROVED** [review]

**Pull request merged**